src/com/yetanalytics/lrs_admin_ui/language.cljs

```diff
@@ -66,6 +66,11 @@
    :browser.query {:en-US "Current Query:"}
    :browser.key-note {:en-US "Please Choose an API Key Above to Browse LRS Data"}
    :browser.refresh {:en-US "Refresh"}
+   :browser.filters {:en-US "Filters:"}
+   ;;CSV Download 
+   :csv-property-paths.title {:en-US "CSV Headers"}
+   :csv.property-paths.add {:en-US "Add Header "}
```

Comment:
Please change these to `CSV Columns` / `Add CSV Column`

Reply:
Changed text in 309095a6fc5bf19092c22660f707e4fba76653f4